experiment-rollout.js

```diff
@@ -161,7 +181,7 @@ const data = {
         notes: [
             {
                 title: 'Masked Links',
-                text: 'Current rollout does not include **masked links**.'
+                text: 'Currently, rollouts do not include **masked links**.'
```

Comment:
```suggestion
                text: 'Currently, rollouts does not include **masked links**.'
```
shouldn't it be "does" ?

Reply:
Nope, it should be do.

Rollouts does not isn't grammatically correct.